atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
For help_file, it appears to follow the same path, but it uses the `metadata_fragment` data structure which is already handled by `orchestrate_build`.

It could be a bad case of Mondays, but I'm not really seeing how the help_file key is actually added to `metadata_fragment` :(